test/babel/fixtures/issue-427/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I'll do gladly. But, I think I can do after the 2nd week of the next month.(I have to finish my graduate thesis.) Could you wait about 3 weeks?